pkg/store/gcworker/gc_worker.go

```diff
@@ -1212,13 +1208,13 @@ func (w *GCWorker) checkUseDistributedGC() bool {
 
 func (w *GCWorker) resolveLocks(
```

Comment:
Nitpick: the function is long and I would personally prefer splitting it into multiple smaller ones.